data/faq.html

```diff
@@ -1062,6 +1272,29 @@ <h3 lang="es">Búsqueda</h3>
 <p></p>
 Nota: solamente puedes encontrar los sitios que han sido mapeados en 
 <a href="http://www.openstreetmap.org">OpenStreetMap</a>.</dd>
+<dd lang="fr">Cliquer sur l'icone de recherche dans l'écran principal. Vous pouvez rechercher en
+utilisant l'un des cas suivants :
+<p></p>
+<b><em>Adresse</em></b> : entrez le N° de la maison, le nom de la rue. Vous pouvez également inquer
+le nom de la ville (si vous ne trouvez toujours pas) et le code postal.
+Par exemple: 41, Rue des Alouettes 75000 Paris.
+<p></p>
+<b><em>Nom d'un lieu</em></b> : Par exemple : Place de la Mairie.
+<p></p>
+<b><em>Mots clefs</em></b> : Les catégories les plus populaires sont visibles par des icones dans
+le feuillet de recherche, mais vous pouvez aussi utiliser des mots clefs comme camping, parc à vélo etc...
```

Comment:
s/parc/parking/

Or change data/categories.txt if you see your variant more appropriate.